homeassistant/components/samsungtv/__init__.py

```diff
@@ -92,6 +96,18 @@ async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
     if DOMAIN not in config:
         return True
 
+    ir.async_create_issue(
+        hass,
+        DOMAIN,
+        "deprecated_yaml",
+        breaks_in_ha_version="2023.6.0",
+        is_fixable=False,
+        severity=ir.IssueSeverity.WARNING,
+        translation_key="deprecated_yaml",
+        translation_placeholders={
+            "on_action_url": "https://www.home-assistant.io/integrations/samsungtv/#turn-on-action"
+        },
```

Comment:
There is also a learn more URL to set:

```suggestion
        translation_placeholders={
            "on_action_url": "https://www.home-assistant.io/integrations/samsungtv/#turn-on-action"
        },
        learn_more_url="https://www.home-assistant.io/integrations/samsungtv/#turn-on-action",
```